CHANGELOG.md

```
@@ -1,3 +1,46 @@
+* Move OPNsense services functions to REST API by [@Snuffy2](https://github.com/Snuffy2) in https://github.com/travisghansen/hass-opnsense/pull/206
+### 🎉 Enhancements & New Features 🎉
+* Move HA services from entity level to platform level by [@Snuffy2](https://github.com/Snuffy2) in https://github.com/travisghansen/hass-opnsense/pull/204
+* Add Unbound Blocklist Switch by [@Snuffy2](https://github.com/Snuffy2) in https://github.com/travisghansen/hass-opnsense/pull/210
+* Sort Device Tracker Selector by [@Snuffy2](https://github.com/Snuffy2) in https://github.com/travisghansen/hass-opnsense/pull/214
+* Add concurrency to Coordinator by [@Snuffy2](https://github.com/Snuffy2) in https://github.com/travisghansen/hass-opnsense/pull/218
+* Optimize switches with concurrency and update logic by [@Snuffy2](https://github.com/Snuffy2) in https://github.com/travisghansen/hass-opnsense/pull/219
+### Other Changes
+* Turn down Debug logging by [@Snuffy2](https://github.com/Snuffy2) in https://github.com/travisghansen/hass-opnsense/pull/211
+* Remove unused get_interfaces by [@Snuffy2](https://github.com/Snuffy2) in https://github.com/travisghansen/hass-opnsense/pull/216
+
+## New Contributors
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)